src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,26 @@
-export const TodoFilter = () => (
+import React from 'react';
+import { Status } from '../../types/Status';
+
+type Props = {
+  currentQuery: string;
+  onSetQuery: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onClearQuery: () => void;
+  onSelectedStatus: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  currentQuery,
+  onSetQuery,
+  onClearQuery,
+  onSelectedStatus,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
+        <select data-cy="statusSelect" onChange={onSelectedStatus}>
+          <option value={Status.All}>All</option>
+          <option value={Status.Active}>Active</option>
+          <option value={Status.Completed}>Completed</option>
```

Comment:
You can get an array of values from your enum with `Object.values` and then render them with map